snap/snapcraft.yaml

```
@@ -23,11 +28,19 @@ apps:
       - network-bind
 
 parts:
-  pollen:
-    plugin: go
+  aproxy:
+    plugin: nil
     source: .
     build-snaps:
       - go
+    override-build: |
+      snapcraftctl build
+      GOARCH=$SNAPCRAFT_TARGET_ARCH
+      CGO_ENABLED=0
```

```suggestion
      export GOARCH=$SNAPCRAFT_TARGET_ARCH
      export CGO_ENABLED=0
```
Oops, sorry about this.